wolox-books/src/app/screens/home/home.component.scss

```diff
@@ -0,0 +1,52 @@
+@import '../../../scss/variables';
+@import '../../../scss/mixins';
+@import '../../../scss/global-classes';
+
+.book {
+  @include card-shadow;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 32px 20px 32px;

```

Comment:
You can use a shorthand like `padding: 32px 32px 20px;`

Reply:
Cool ☝️